src/main/java/tk/ynvaser/quiz/model/users/Role.java

```diff
@@ -0,0 +1,13 @@
+package tk.ynvaser.quiz.model.users;
+
+public enum Role {
+    ;
+    public static final String ADMIN = "admin";
+    public static final String QUIZ_MASTER = "quizMaster";
+    public static final String TEAM_LEADER = "teamLeader";
+    public static final String USER = "user";
+
+    public static String[] getAllRoles() {
+        return new String[]{QUIZ_MASTER, TEAM_LEADER, USER, ADMIN};
```

Comment:
There is rarely a good reason to return an array rather than a List.
In this instance, you could just use Role.values():
https://www.baeldung.com/java-enum-iteration

Reply:
Class removed for now